packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -526,6 +529,7 @@ class BallisticScrollActivity extends ScrollActivity {
     super.delegate,
     Simulation simulation,
     TickerProvider vsync,
+    this.shouldIgnorePointer,
```

Comment:
This might be a breaking change, let me run some internal tests first. One way to work around this is to make it an optional parameter with a default behavior.

Reply:
Actually I was able to do a quick look up and found no uses of this class internally. ✅

Reply:
I did find a breakage in an external package while testing this with my existing code, but they were only overriding `BallisticScrollActivity` to perform the exact same function as here.